makefile

```diff
@@ -0,0 +1,16 @@
+.PHONY: build test lint
+
+VERSION=$(shell git describe --tags --dirty --always)
+
+build:
+	go build -ldflags "-X 'github.com/conduitio-labs/conduit-connector-sql-server.version=${VERSION}'" -o conduit-connector-sql-server cmd/connector/main.go
+
+test:
+	go test $(GOTEST_FLAGS) ./...
+
+lint:
+	golangci-lint run
+
+mockgen:
+	mockgen -package mock -source destination/interface.go -destination destination/mock/destination.go
```

Comment:
can we also add these command to the file that has the interface? at the beginning of the file, like:
` //go:generate mockgen ...`

Reply:
I added comment how generate mock near interface declaration

Reply:
I meant something like this https://github.com/ConduitIO/conduit/blob/e446e1c5e43319395151fdcc06b8726e0f7fcac5/pkg/orchestrator/orchestrator.go#L15 , we usually have the mock command near the interface, so when you change the interface, you would just click the run button near the `go:generate` command, and regenerate the mocks

Reply:
ok. I have done it